app/views/projects/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<h1>Home Page</h1>
+  <%= link_to "Projects", "/projects/show" %><br>
```

Comment:
This link is also misleading since a show page will require a specific Id... Going to check on your routes...

Reply:
I see now - your routes don't have a path to `projects/show`; but there is a controller-action pairing that would work. If you didn't try it, run `rails s` and see if you can click this link - it shouldn't work because we want to put the exact path (URL) to a specific project. It would be something like `/projects/2` or whatever ID exists in the database for a specific project instance.